controllers/activegate/reconciler/capabilityr/reconciler.go

```diff
@@ -1,11 +1,15 @@
-package capability
+package capabilityr
```

Comment:
Package rename on purpose? "capabilityr"?

Reply:
It was a middle step during refactoring. I missed it before commit.